.github/workflows/x-keycloak-operator-hub-publish.yml

```diff
@@ -0,0 +1,155 @@
+name: X Keycloak Operator Hub publish
+
+on:
+  workflow_call:
+    inputs:
+      gh-org:
+        required: true
+        type: string
+      quay-org:
+        required: true
+        type: string
+      community-operators-repo:
+        required: true
+        type: string
+      prod-operators-repo:
+        required: true
+        type: string
+      mvn-url:
+        required: true
+        type: string
+      version:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+    secrets:
+      GH_TOKEN:
+        required: true
+      MVN_USERNAME:
+        required: true
+      MVN_TOKEN:
+        required: true
+      QUAY_USERNAME:
+        required: true
+      QUAY_TOKEN:
+        required: true
```

Comment:
Do we need quay and mvn in operatorhub releases?

Reply:
I've revised that, and we don't need to use login to the Quay. Regarding the Maven, I've been following the same approach as in the build phase of `x-keycloak-operator.yml`. In this case, I'd keep it as is in order to keep consistency between those; I think it's not such a crucial point, right?

Reply:
I think we don't need to care too much about consistency here as `x-keycloak-operator` is something different from operatorhub release. :) My point was that we should be rather conservative when it comes to passing Secrets into flows – pass just the bare minimum we need. But it's not a blocker from my perspective.

Reply:
Ok, thanks